frontend/pages/topics/[slug].tsx

```diff
@@ -3,9 +3,23 @@ import ModuleContainer from "@/components/details/ModuleContainer";
 import Layout from "@/components/Layout";
 import PageHeader from "@/components/PageHeader";
 import { Topic } from "@/types/modules";
+import { Card, CardContent, CardHeader } from "@material-ui/core";
+import { styled } from "@material-ui/core/styles";
 import { GetStaticPaths, GetStaticProps } from "next";
 import Link from "next/link";
-import { FC } from "react";
+import React, { FC } from "react";
+import HTMLEllipsis from "react-lines-ellipsis/lib/html";
+
+const StyledTopicResultCard = styled(Card)({
```

Comment:
Nit: can we rename `StyledTopicResultCard` to `PropositionCard` or `TopicPropositionCard`? IMO it's best to prefix with `Styled` when it's not convenient to think of a more descriptive name (e.g. [`<StyledPagination/>`](https://github.com/ModularHistory/modularhistory/blob/2337d77001d0af0d25fadce3e5a7e3c9b6d13cba/frontend/components/Pagination.tsx#L7)), but otherwise `Styled` doesn't need to be in the name (e.g. [`<TagChip/>`](https://github.com/ModularHistory/modularhistory/blob/2337d77001d0af0d25fadce3e5a7e3c9b6d13cba/frontend/components/topics/Tag.tsx#L7)).